src/pages/products/products.js

```diff
@@ -23,36 +32,15 @@ export const Products=()=>{
     const data5=priceRange(data4,state)
     const data6=rating(data5,state)
     const data7=categoryHandler(data6,state)
-    const compose=data7
+    const data8=clearAll(data7,state)
+    const compose=data8
```

Comment:
i think its not a good naming convention looks odd to me

Reply:
yes will change it